tools/mason/MasonNew.chpl

```diff
@@ -46,20 +46,30 @@ proc masonNew(args: [] string) throws {
                                  opts=["--name"]);
 
   var showFlag = parser.addFlag(name="show", defaultValue=false);
+  var appFlag = parser.addFlag(name="app", defaultValue=false);
+  var libFlag = parser.addFlag(name="lib", defaultValue=false);
+  var lightFlag = parser.addFlag(name="light", defaultValue=false);
   var dirArg = parser.addArgument(name="directory", numArgs=0..1);
 
   parser.parseArgs(args);
 
   var vcs = !vcsFlag.valueAsBool();
   var show = showFlag.valueAsBool();
+  var isApplication = appFlag.valueAsBool();
+  var isLibrary = libFlag.valueAsBool();
+  var isLightweight = lightFlag.valueAsBool();
+
   var packageName = '';
   var dirName = '';
   var version = '';
   var chplVersion = '';
   var license = 'None';
+  // Default created mason project is a package (has a main func)
+  var packageType = 'app';
```

Comment:
It looks like you use "application" in other places, does "app" have special meaning or should this be application?

Reply:
Oof, good catch.